tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -1481,6 +1481,15 @@ var _ = ginkgo.Describe("TiDBCluster", func() {
 		err = oa.WaitForTidbClusterReady(fromTc, 30*time.Minute, 5*time.Second)
 		framework.ExpectNoError(err, "Expected TiDB cluster ready")
 
+		ginkgo.By("Update cdc config to use config file")
```

Comment:
Suggest checking the config file here, refer to https://github.com/pingcap/tidb-operator/blob/master/tests/e2e/tidbcluster/tidbcluster.go#L1667-L1677.

Reply:
https://github.com/pingcap/tidb-operator/pull/4010/commits/a61ed34931c2491cfbde3ef374dda0bc527ef3c0